### DISCUSSION TITLE
html aparece en blanco en navegador

### DISCUSSION THREAD

**Discussion by DavidMain9798** [Launch-X-Latam/MisionFrontEnd#278] [General]
Buen dia explorers tengo un problema con mi practica de la taqueria, sucede que ya hice todo el codigo pero al abrir el html en el navegador, no sale nada solo aparece todo blanco sin contenido, al principio aparecía solo el titulo que le puse a mi taqueria pero luego aun teniendo todo el código escrito con imagen y todo tal como lo hizo el comander, intente todo, hasta cerre el visual estudio code, hasta copie y pegue el código de los proyectos de otros compañeros en un nuevo archivo.html y al intentar abrirlo pasa lo mismo, no muestra nada, todo en blanco pero, incluso cambie de navegador a opera y sale igual, también busque el clásico "hola mundo" en Internet copie todo el código en un nuevo archivo.html y salio igual en blanco todo, gracias de antemano. 

pd: uso chrome, y ya instale el plugin go live en vsc

**Comment by GeraldinTorres** [comment]
Hola tuve el mismo problema esta es la solucion que me dieron 

https://github.com/Launch-X-Latam/MisionFrontEnd/discussions#discussioncomment-3999394

**Reply by DavidMain9798** [reply]
Muchas gracias, a usted si le funciono?

**Reply by GeraldinTorres** [reply]
Hola si claro ya todo me funciona
